process.env.npm_config_user_agent is undefined nodejs v12, npm 7.7.0 @ arch

Hi,

i've use you npm plugin on arch with nodejs v12.21.0 and npm v 7.7.0. It's looks like arch compile npm without user_agent property.

If I try to run tests. I get a undefined error:
```
2. Test (Generic) runs transitioning to OS specific code
/home/denco/Projects/public/run-script-os/index.js:65
const isYarn = process.env.npm_config_user_agent.includes('yarn');
                                                 ^

TypeError: Cannot read property 'includes' of undefined
    at Object.<anonymous> (/home/denco/Projects/public/run-script-os/index.js:65:50)
    at Module._compile (internal/modules/cjs/loader.js:999:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1027:10)
    at Module.load (internal/modules/cjs/loader.js:863:32)
    at Function.Module._load (internal/modules/cjs/loader.js:708:14)
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:60:12)
    at internal/main/run_main_module.js:17:47
npm ERR! code 1
npm ERR! path /home/denco/Projects/public/run-script-os
npm ERR! command failed
```

thx!

--- a/index.js
+++ b/index.js
@@ -62,7 +62,7 @@ if (!(options[0] === "run" || options[0] === "run-script")) {
 options[1] = expandShorthand(options[1]);
 
 // Check for yarn without install command; fixes #13
-const isYarn = process.env.npm_config_user_agent.includes('yarn');
+const isYarn = (process.env.npm_config_user_agent && process.env.npm_config_user_agent.includes('yarn')) ? true : false;
 if (isYarn && !options[1]) options[1] = 'install';
 
 let osCommand = `${options[1]}:${platform}`;

